packages/mongo-nav/tsconfig.json

```diff
@@ -25,6 +25,9 @@
     {
       "path": "../icon"
     },
+    {
+      "path": "../testing-lib"
```

Comment:
stack diffs 🙈

Reply:
😮 did I miss something? this PR introduces the mongo-nav changes, so it seems like the right place to update the mongo-nav tsconfig

Reply:
Oh, this IS the stacked diff. My bad! Thought this was into master.